projects/npm-tools/packages/npm-scripts/src/webpack/createFederationConfig.js

```diff
@@ -92,16 +98,12 @@ module.exports = async function () {
 		}, {}),
 	};
 
-	createTempFile(
-		'federation.plugin.config.json',
-		JSON.stringify(federationPluginConfig, null, 2),
-		{autoDelete: false}
-	);
-
 	return {
 		context: process.cwd(),
 		devtool: 'source-map',
-		entry: nullJsFilePath,
+		entry: {
```

Comment:
This is an entry point that, because is called the same as the library, signals webpack to use it as initialization code.

See [this webpack issue](https://github.com/webpack/webpack/issues/12834) for more information.

Reply:
Added this :point_up: to the commit message for the future generations.